easylist_adult/adult_thirdparty.txt

```diff
@@ -246,6 +246,7 @@
 ||widget.faphouse.com^$third-party
 ||widgets.comcontent.net^
 ||wp-script.com/img/banners/
+||x666.io^$third-party
```

Comment:
You should move this one to at least https://github.com/easylist/easylist/blob/master/easylist_adult/adult_adservers.txt even I actually  personally would prefer the https://github.com/easylist/easylist/blob/master/easylist/easylist_adservers.txt

Own notes about this hosts: https://mypdns.org/my-privacy-dns/matrix/-/issues/1926

Reply:
I no more see the request and an element has changed.

Reply:
> I no more see the request and an element has changed.

Ok, I've just visited the site a couple of hours ago, and there the x666 was requested :wink: :confused: